code/modules/power/apc.dm

```diff
@@ -65,27 +65,27 @@
 	var/start_charge = 90				// initial cell charge %
 	var/cell_type = 2500	//Base cell has 2500 capacity. Enter the path of a different cell you want to use. cell determines charge rates, max capacity, ect. These can also be changed with other APC vars, but isn't recommended to minimize the risk of accidental usage of dirty editted APCs
 	var/opened = 0 //0=closed, 1=opened, 2=cover removed
-	var/shorted = 0
+	var/shorted = FALSE
 	var/lighting = 3
 	var/equipment = 3
 	var/environ = 3
-	var/operating = 1
-	var/charging = 0
+	var/operating = TRUE
+	var/charging = 0 // 0 is not charging, 1 is charging, 2 is fully charged
```

Comment:
Could you make this use defines for the states?

Reply:
I've made opened, charging and electronics_state (renamed from has_electronics) use defines for states.
I have not converted autoflag to use defines for states, as I'm not confident I understand what it is doing when it has a value set outside the range that it's supposed to have...

Reply:
I'm not sure why my commit got named 'eye_protection ='...